cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/ExecutableBuilder.java

```diff
@@ -83,10 +82,15 @@
 
     private List<Transformer> preExecTransformers;
     private List<Transformer> postExecTransformers;
-    private List<String> execAdditionalKeywords = Arrays.asList(SlangTextualKeys.ACTION_KEY, SlangTextualKeys.WORKFLOW_KEY, SlangTextualKeys.EXECUTABLE_NAME_KEY);
+    private List<String> execAdditionalKeywords = Arrays.asList(
+            SlangTextualKeys.JAVA_ACTION_KEY,
+            SlangTextualKeys.PYTHON_ACTION_KEY,
+            SlangTextualKeys.WORKFLOW_KEY,
+            SlangTextualKeys.EXECUTABLE_NAME_KEY
```

Comment:
Don't we want to add also the additional keywords of class_name, method_name, script, etc... ?